sql/core/src/test/scala/org/apache/spark/sql/connector/InsertIntoTests.scala

```diff
@@ -446,8 +446,7 @@ trait InsertIntoSQLOnlyTests
       }
     }
 
-    test("InsertInto: overwrite - multiple static partitions - dynamic mode") {
-      // Since all partitions are provided statically, this should be supported by everyone
+    dynamicOverwriteTest("InsertInto: overwrite - multiple static partitions - dynamic mode") {
       withSQLConf(PARTITION_OVERWRITE_MODE.key -> PartitionOverwriteMode.DYNAMIC.toString) {
```

Comment:
Could you remove this line because `dynamicOverwriteTest` has it already?

Reply:
Done.